internal/lock.go

```diff
@@ -26,6 +25,12 @@ type config struct {
 	Resource []Resource
 }
 
+type resource struct {
```

Comment:
What's this? You're almost duplicating an existing struct...?

Reply:
You've marked this as resolved... but the struct is still here... so it's not resolved.